### PR TITLE
RHCOS live ISO

### PR DESCRIPTION
**Pull request by rollandf**

### CODE REVIEW

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -238,7 +238,7 @@ func (b *bareMetalInventory) getUserSshKey(params installer.GenerateClusterISOPa
 		return ""
 	}
 	return fmt.Sprintf(`{
-		"name": "core",
+		"name": "systemUser",
```
Comment (romfreiman) [inline internal/bminventory/inventory.go:241]: no. it should be core.

File: internal/bminventory/inventory.go [hunk 2]
```diff
@@ -63,7 +63,7 @@ var (
 )
 
 type Config struct {
-	ImageBuilder        string `envconfig:"IMAGE_BUILDER" default:"quay.io/ocpmetal/installer-image-build:stable"`
+	ImageBuilder        string `envconfig:"IMAGE_BUILDER" default:"quay.io/yshnaidm/installer-image:rchos"`
```
Comment (romfreiman) [inline internal/bminventory/inventory.go:66]: why do u revert

Comment (rollandf) [reply internal/bminventory/inventory.go:66]: This is for testing with RHCOS

File: internal/bminventory/inventory.go (hunk 1 above)
Comment (rollandf) [reply internal/bminventory/inventory.go:241]: core already exists. Not sure how to update an existing user.

Comment (romfreiman) [reply internal/bminventory/inventory.go:241]: mmm worth checking. systemUser is missleading, that's why we aligned to core all over

**Pull request closed without merging**